### ISSUE TITLE
How to remove W/System.err: Aug 21, 2020 4:33:38 PM com.github.aakira.napier.DebugAntilog performLog

### ISSUE THREAD

**Issue by schmidt9** [AAkira/Napier#48]
Hello
How to remove repeating message in Console and LogCat in Android Studio?
> W/System.err: Aug 21, 2020 4:33:38 PM com.github.aakira.napier.DebugAntilog performLog

**Comment by AAkira** [comment]
@schmidt9 How was this?

**Comment by AcoustickSan** [comment]
@schmidt9 add this lines to you android gradle file
```
    configurations.all {
        resolutionStrategy {
            exclude("com.github.aakira", "napier-jvm")
        }
    }
```

**Comment by AAkira** [comment]
Thanks.
This is another solution.

**Comment by schmidt9** [comment]
> I see.
> I think it is for JVM.
> 
> Do you write the `implementation "com.github.aakira:napier-android:1.4.0` in the Android module?
> I think you use `DebugAntilog` for JVM.

I have `implementation "com.github.aakira:napier:$napierVersion"` in `commonMain` only as you suggested in  #47

**Comment by schmidt9** [comment]
> @schmidt9 add this lines to you android gradle file
> 
> ```
>     configurations.all {
>         resolutionStrategy {
>             exclude("com.github.aakira", "napier-jvm")
>         }
>     }
> ```

@AcoustickSan where should I  add it properly? I have a gradle script with multiple targets, whenever I add you snippet it gives me 
```
Gradle DSL method not found: 'exclude()'
Possible causes:
The project 'KotlinNativeTest' may be using a version of the Android Gradle plug-in that does not contain the method (e.g. 'testCompile' was added in 1.1.0).
Upgrade plugin to version 4.0.1 and sync project
```
(while I've already got Gradle 4.0.1)

**Comment by schmidt9** [comment]
> I'm sorry.
> I couldn't reproduce it in my environment.
> 
> Do you have example code or repository?
> I'll check it.

BTW the same with `performLog` you have on this screenshot **https://github.com/AAkira/Napier/blob/master/arts/screenshot-jvm.jpg**

**Comment by AAkira** [comment]
@schmidt9 

I published the Napier`1.4.1-alpha1`.

> I have implementation "com.github.aakira:napier:$napierVersion" in commonMain only as you suggested in #47

You should write this `implementation "com.github.aakira:napier:1.4.1-alpha1` in this [line](https://github.com/AAkira/Napier/blob/master/android/build.gradle#L40).

And maybe you don't need to write exclude codes.
Please check it.

**Comment by schmidt9** [comment]
@AAkira now it seems to be OK, `performLog` line gone
